optimizely/entities.py

```diff
@@ -109,7 +109,8 @@ def __init__(self, id, value, **kwards):
       self.id = id
       self.value = value
 
-  def __init__(self, id, key, variables=None, **kwargs):
+  def __init__(self, id, key, featureEnabled=False, variables=None, **kwargs):
```

Comment:
Actually, just realized this might not work. Only feature experiments will have this property. Regular experiment variations will omit this property and you will run into a situation where the variation comes in with just `id, key, and variables` so you'd need to accommodate that scenario

Reply:
Shouldn't be an issue. generate_key_map in project_config calls as key,value arguments. 
Still, have added a new test case to test the scenario.